### PR TITLE
Use space instead of `;` as a path separator

### PR DESCRIPTION
**Pull request by jakubfijalkowski**

### CODE REVIEW

**Review by shilangyu — APPROVED** [review]

**Review by lukaszgarstecki — APPROVED** [review]

**Comment by Saancreed** [comment]
Does it just consume arguments from command like as split by the OS? Or would I need to call it like `-p 'a.csproj b.csproj'`?

**Comment by jakubfijalkowski** [comment]
@Saancreed both.

**Review by Saancreed — APPROVED** [review]

File: src/LeanCode.ContractsGenerator/Program.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public class ProjectOptions : IOptions
     {
         public string OutputFile { get; set; }
 
-        [Option('p', "project", Required = true, Separator = ';', MetaValue = "FILE", HelpText = "The project file with contracts. To pass multiple projects, separate the paths with ';'.")]
+        [Option('p', "project", Required = true, MetaValue = "FILE", HelpText = "The project file with contracts. To pass multiple projects, separate the values with space.")]
```
Comment (konowrockis) [inline src/LeanCode.ContractsGenerator/Program.cs:22]: How does it work when you have paths with spaces? Is it like `--project "../path with spaces/1.csproj" "../path with spaces/2.csproj"`?

Comment (shilangyu) [reply src/LeanCode.ContractsGenerator/Program.cs:22]: > as split by the OS.
yes

Comment (jakubfijalkowski) [reply src/LeanCode.ContractsGenerator/Program.cs:22]: @konowrockis Exactly that. Each parameter as interpreted by the OS parser will be considered a separate path.

**Review by konowrockis — APPROVED** [review]

**Pull request merged**